lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -364,7 +366,11 @@ <h4 class="message-title">${_('Almost there!')}</h4>
                 % endif
               </div>
               <div class="verification-cta">
-                <a href="${reverse('verify_student_verify_now', kwargs={'course_id': six.text_type(course_overview.id)})}" class="btn" data-course-id="${course_overview.id}">${_('Verify Now')}</a>
+                % if redirect_to_idv_microfrontend():
+                  <a href="${settings.ACCOUNT_MICROFRONTEND_URL}/id-verification?${six.text_type(course_overview.id)}" class="btn" data-course-id="${course_overview.id}">${_('Verify Now')}</a>
```

Comment:
Why not wrap this logic into a python class function in the `IDVerificationService` in the `lms.djangoapps.verify_student.services` module? This way, that function can be used here and in the date_summary.py file.

Reply:
Implemented it that way instead, thank you!